src/main/java/Jelly.java

```diff
@@ -0,0 +1,183 @@
+import java.util.*;
+//enum is not required
+public class Jelly {
+
+    private static TaskList list;
+    private static String line = "\n-------------------------------------------";
+
+    private static String welcome = "(ᵔ_ᵔ) Hello! I'm Jelly\nWhat can I do for you?";
+    private static String farewell = "(•́︿•̀) Bye. Hope to see you again soon!";
+
+    private static String path = "jelly.txt";
+    public static void main(String[] args) {
+
+        list = new TaskList(path);
+
+        System.out.println(line);
+        System.out.println(welcome);
+        System.out.println(line);
+
+        Scanner scanner = new Scanner(System.in);
+
+        while(!Read(scanner).equals("bye"));
+
+        System.out.println(farewell);
+        System.out.println(line);
+
+        list.save(path);
+    }
+
+    public static String Read(Scanner scanner){
+
+        String message = scanner.nextLine();
+        String[] lines = message.split("\\s+");
+        String command = lines[0];
+
+        String argument = "";
+
+        if(lines.length > 1) {
+        argument = message.substring(command.length() + 1);
+        }
+
+        System.out.println(line);
+
+        switch(command){
```

Comment:
Refering to the coding standard for switch commands the identations are different from what we might be used to. The lines of 'case' statements should be on the same identation level as the 'switch' statement itself. Only the items within the cases needs to be further indented